queries/util/query.js

```diff
@@ -0,0 +1,20 @@
+const { exec } = require("child_process");
+exports.query = function query(query) {
+  const cmd = `bq query --nouse_legacy_sql '${query}'`;
```

Comment:
The SQL in this patch would be more readable if dataset were injected here as a command line parameter of `bq`